lib/deviceSpecs/specifications.js

```diff
@@ -34,13 +35,14 @@ module.exports = {
 		},
 		serial: {
 			vid: '1d50',
-			pid: '607d'
+			pid: '607d',
+			serialNumber: 'Spark_Devices_Spark_Core'
```

Comment:
Based on https://github.com/spark/firmware/blob/develop/hal/src/core/usb_desc.c it looks like newer Core firmware might report as `Particle_Spark_Core_with_WiFi` so maybe we should reduce this to `Spark_Core`?